llvm/lib/CheerpUtils/CMakeLists.txt

```diff
@@ -28,6 +28,10 @@ add_llvm_component_library(LLVMCheerpUtils
   StoreMerging.cpp
   CheerpLowerInvoke.cpp
   SinkGenerator.cpp
+  PartialExecuter.cpp
```

Comment:
Merge this commit with the actual one adding the .cpp file, otherwise we have a state which will not build

Reply:
By the way, I am quite sure we had to move PreExecute into CheerpWriter due to weird dependencies issues. Please move PartialExecuter there as well.